test/Observable_stock_test.dart

```diff
@@ -0,0 +1,47 @@
+library wCommon.test;
+
+// Testing imports
+import 'package:unittest/unittest.dart';
+import 'package:mock/mock.dart';
+import 'package:unittest/html_enhanced_config.dart';
+
+// Testing dependencies
+import 'dart:async';
+import 'dart:js';
+import 'package:wCommon/wCommon.dart';
+
+class MockJsObject extends Mock {}
+
+main() {
+
+  useHtmlEnhancedConfiguration();
+
+  var observable;
+  var jsObservable;
+
+  group('Observable', () {
+
+    setUp(() {
+      jsObservable = new MockJsObject();
+
+      observable = new Observable(jsObservable);
+    });
+
+    test('it should return a stream for the stream property', () {
+      var stream = observable.stream;
+
+      expect(stream, new isInstanceOf<Stream>());
+    });
+
+    test('it should proxy calls to the dispatch method down to the jsObservable', () {
+      var params = [];
+      var callback = () {};
+      var mockReturnValue = 'dispatch called';
+      jsObservable.when(callsTo('callMethod', 'dispatch')).alwaysReturn(mockReturnValue);
+
+      observable.dispatch(params, callback);
+
+      jsObservable.getLogs(callsTo('callMethod'), returning(mockReturnValue)).verify(happenedExactly(1));
```

Comment:
First impression is that I did not like the stock mocking.  It feels needlessly verbose for simple tasks.  It takes this whole line, plus line 40, to even be able to tell when a mock was called.